packages/teams-js/src/private/conversations.ts

```diff
@@ -155,7 +155,7 @@ export namespace conversations {
         registerHandler(
           'startConversation',
           (subEntityId: string, conversationId: string, channelId: string, entityId: string) =>
-            openConversationRequest.onStartConversation({
+            openConversationRequest.onStartConversation?.({
```

Comment:
>openConversationRequest.onStartConversation?. [](http://example.com/codeflow?start=12&length=45)

Even though the code above this appears to try to handle this case by doing `if (openConversationRequest.onStartConversation)`, that is unfortunately not sufficient for this pattern because the in the call to `registerHandler`, an arrow function is used as the second parameter, which is creating a new inline function that is calling `openConversationRequest.onStartConversation`. Between this point when `registerHandler` is called and when that arrow function is invoked, `openConversationRequest.onStartConversation` could become `undefined` (that arrow function may not be executed until some time way in the future after many things have happened).

Using the `?.` operator ("optional chaining operator") here will cause this function to be called if it exists and will cause nothing to happen in the function is `undefined`. No other special consideration is needed since the function returns `void` and it won't matter if it doesn't execute (or rather, it won't matter if _there's nothing to execute_)